src/server/utils/fuseapi.js

```diff
@@ -0,0 +1,43 @@
+import request from 'request'
+import qs from 'qs'
+class FuseApi {
+  constructor() {
+    this.mainUrl = 'https://explorer.fuse.io'
+  }
+
+  /**
+     * Get transaction list
+     * @param {
+        address,
+        page,
+        offset,
+        filterby,
+        startBlock,
+        endBlock,
+        starttimestamp,
+        endtimestamp,
+        sort
+     * } options 
+     */
+  getTxList(options) {
+    return this.runMethod('account', 'txlist', options)
+  }
+
+  runMethod(module, action, options) {
+    return new Promise((resolve, reject) => {
+      const queryParams = qs.stringify({
+        module,
+        action,
+        ...options
+      })
+      request(`${this.mainUrl}/api?${queryParams}`, { json: true }, (err, res, body) => {
```

Comment:
use the frameworks we already use, such as crossfetch/axios, whatever we use in other places in the backend

Reply:
we don't use crossfetch and axios yet. ok I will install axios

Reply:
done

Reply:
i requested whatever we use on the backend.

Reply:
We don’t use any of frameworks to send requests in any places in the backend
that’s why I used request first time.
then replace with axios

Reply:
done